aspnetcore/mvc/razor-pages/index.md

```diff
@@ -204,6 +204,38 @@ The `OnPostDeleteAsync` method:
 * If the customer contact is found, they're removed from the list of customer contacts. The database is updated.
 * Calls `RedirectToPage` to redirect to the root Index page (`/Index`).
 
+::: moniker range=">= aspnetcore-2.1"
+## Manage HEAD requests with the OnGet handler
+
+Ordinarily, a HEAD handler is created and called for HEAD requests:
+
+```csharp
+public void OnHead()
+{
+    HttpContext.Response.Headers.Add("HandledBy", "Handled by OnHead!");
+}
+```
+
+If no HEAD handler (`OnHead`) is defined, Razor Pages falls back to calling the GET page handler (`OnGet`) in ASP.NET Core 2.1 or later. This behavior is opt-in with the [SetCompatibilityVersion method](xref:fundamentals/startup#setcompatibilityversion-for-aspnet-core-mvc) in `Startup.Configure` for ASP.NET Core 2.1&rarr;2.x:
```

Comment:
Guess it's a bit verbose to call this out, but precise.

Reply:
I suggest that if we *do do it* ... we'll probably have less trouble with adoption/migration.

We're probably going to get slammed by reader remarks now that they go directly into docs repo issues. I humbly beseech the :pray: server gods :pray: for this approach to work and save us a few reader feedback issues of *Why isn't this working!?*